config/urls.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib import admin
+from django.urls import path, include
+from rest_framework import permissions
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Mobile Bidding Platform",
+        default_version='v1',
+        description="API documentation for Mobile Bidding Platform",
+    ),
+    public=True,
+    permission_classes=[permissions.AllowAny],
+)
+
+
+urlpatterns = [
+    path('swagger/', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui'),
+    path('admin/', admin.site.urls),
+    path('api/users/', include("mobile_marketplace.users.api.urls")),
+    path('api/mobiles/', include("mobile_marketplace.mobiles.api.urls")),
+    path('api/bids/', include("mobile_marketplace.bids.api.urls"))
```

Comment:
Its best practice to add api versioning. `path('api/v1/users/', include("mobile_marketplace.users.api.urls")),`